e2e/utilities/utils.ts

```diff
@@ -23,14 +23,77 @@
  * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { browser, protractor, ElementFinder, ExpectedConditions as EC, by, logging } from 'protractor';
+import { browser, protractor, ElementFinder, ExpectedConditions as EC, by, logging, until } from 'protractor';
 import { Logger } from '@alfresco/adf-testing';
 import { BROWSER_WAIT_TIMEOUT, E2E_ROOT_PATH, EXTENSIBILITY_CONFIGS } from '../configs';
 
 const path = require('path');
 const fs = require('fs');
 const StreamZip = require('node-stream-zip');
 
+export async function waitCss(css: string, errorMessage?: string): Promise<any> {
+  return browser.wait(
+    until.elementLocated(by.css(css)),
+    BROWSER_WAIT_TIMEOUT,
+    errorMessage || 'Timeout waiting for CSS selector'
+  );
+}
+
+export async function waitForClickable(
```

Comment:
I think these methods related to the elements can be moved into the browser-utils.ts file

Reply:
for now it does not matter, as the idea to move them to ADF later on